### PR TITLE
Fix navigation bar search box bug

### LINKED ISSUE #1549: Small visual glitch in search bar
I suspect this came with the "package button" changes:

CleanShot 2022-02-13 at 06 37 59@2x

The right side of the search box is clipped a bit in dark mode.

### PR DESCRIPTION
**Pull request by daveverwer**
Fixes #1549

Screenshot 2022-02-14 at 16 32 42@2x

### CODE REVIEW

**Review by finestructure — APPROVED** [review]

**Comment by finestructure** [comment]
Let me get the build id stuff merged before this so I don’t need to merge/rebase those three PRs again. I’ll merge this when I’m done with that in the morning!

**Comment by daveverwer** [comment]
👍 No rush

**Pull request merged**
